build.gradle

```diff
@@ -10,6 +10,19 @@ plugins {
 group = "com.incognia"
 version = "2.2.0"
 
+task createProjectVersionFile {
+    doLast {
+        new File("$projectDir/src/main/java/com/incognia/api", "ProjectVersion.java").write(
+                """package com.incognia.api;
+
+public class ProjectVersion {
```

Comment:
I think we can indent these part better, it's very difficult to read whats being created here